db/migrate/20161224170703_create_groups.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGroups < ActiveRecord::Migration[5.0]
+  def change
+    create_table :groups do |t|
+      t.integer :size, null: false, default: 1
+      t.integer :status, null: false, default: 0
+      t.belongs_to :leader, index: true, null: false
```

Comment:
Well, I'm guessing that in some situations the admins would be performing suite selection as well, but yea we're going to want them to assign a leader. We just have to make sure that it's easy to do.